### ISSUE TITLE
Kill event during translation

### ISSUE THREAD

**Issue by littleghostlyfox** [forsatus/Xliff-AI-Translator#2]
I have a kill event without much explanation. The process seems stuck during the translation: after the initial command, the terminal displays the text to translate, then the CPU goes to 80% (13th Gen Intel i5-13600KF @ 5.100GHz), and then Ubuntu seems to automatically end the process. I'm a Linux beginner, so I'm not too sure...

**Comment by forsatus** [comment]
Hello, in terms of power, you have more than enough.
Does the program terminate immediately without translating anything?

**Comment by littleghostlyfox** [comment]
Yes. It displays the initial text to translate, then nothing for a few seconds (~30s?) and a Killed message is displayed.
Thanks for your quick feedback! :)

**Comment by forsatus** [comment]
How much RAM do you have ?

**Comment by littleghostlyfox** [comment]
You should have all infos :) I don't think the problem comes from the hardware capabilities. I tried to see if the code had a Killed function, but I don't think it has one. Some it must be a security of the system that prevents a weird behavior from the code.

![Screenshot from 2024-08-07 09-22-56](https://github.com/user-attachments/assets/ba3aa7ee-9b17-4390-8558-8eb2c9ede502)

**Comment by forsatus** [comment]
Which version of the XLIFF file do you have?

**Comment by littleghostlyfox** [comment]
It's a 1.2 file : 
`<xliff xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 http://docs.oasis-open.org/xliff/v1.2/os/xliff-core-1.2-strict.xsd" version="1.2" xmlns:xhtml="http://www.w3.org/1999/xhtml" xml:space="preserve" xmlns="urn:oasis:names:tc:xliff:document:1.2"><file original="course" datatype="plaintext" source-language="en-US">`

**Comment by littleghostlyfox** [comment]
Hi forsatus! I still have the same problem. Do you have any idea for a fix? :) Thanks!

**Comment by forsatus** [comment]
Hi,

Sorry about this! I’m not sure why it’s behaving this way without providing any feedback. I wasn’t able to reproduce the bug on my end, so it might be something specific to your setup. Could you try running it in a virtual machine or on a fresh OS installation to see if the issue persists?

**Comment by randex** [comment]
Same just happened to me on an M3 Max MacBook Pro with 36 GB of RAM, no way to avoid it, it just shows 
```
zsh: killed     ./xliff_ai_translator   zh
```
at the end.

**Comment by forsatus** [comment]
> Same just happened to me on an M3 Max MacBook Pro with 36 GB of RAM, no way to avoid it, it just shows
> 
> ```
> zsh: killed     ./xliff_ai_translator   zh
> ```
> 
> at the end.

Is it a large file ?

**Comment by randex** [comment]
> Is it a large file ?

746 strings
